src/controller/CHIPDeviceController.cpp

```diff
@@ -753,26 +791,12 @@ void DeviceCommissioner::OnRendezvousStatusUpdate(RendezvousSessionDelegate::Sta
     case RendezvousSessionDelegate::SecurePairingSuccess:
         ChipLogDetail(Controller, "Remote device completed SPAKE2+ handshake\n");
         mRendezvousSession->GetPairingSession().ToSerializable(device->GetPairing());
-
-        if (!mIsIPRendezvous && mPairingDelegate != nullptr)
-        {
-            mPairingDelegate->OnNetworkCredentialsRequested(mRendezvousSession);
-        }
```

Comment:
So basically, all the examples that relies on `OnNetworkCredentialsRequested` are broken ? I guess it affects `CHIPTool iOS` and Android as well ?

Reply:
Yes, basic network commissioning on those apps should be implemented -- "AddWiFiNetwork", "AddThreadNetwork" and "EnableNetwork" need to be implemented on those apps.

Reply:
Ok. So can we try to do 2 separates PR then `BLE as a transport` and `Use Network Commissioning cluster instead of DevicePairingDelegate` ?
Looking at the code it is not 100% clear to me that both needs to happen at the same time, and if they ever do I don't think we can just break working examples.

Reply:
This PR is "BLE as a transport", and it depends on "Use Network Commissioning cluster"

The Python controller can still work after this PR is merged, so iOS & Android controllers should be able to work (no special changes are required for this PR, just add support for network commissioning cluster #5520 ) as well.

Reply:
After discussion with testing team, we will move forward on this PR since TE2 won't test mobile phone apps.

Reply:
> After discussion with testing team, we will move forward on this PR since TE2 won't test mobile phone apps.

@erjiaqing `TE2 won't test mobile phone apps` Does it mean that in TE2 none of the participants will use iOS/Android Chip Tool as a controller and Python controller would be the preferred way ?  Sorry I am not clear about this statement.